docs/ja/faq_misc.md

```diff
@@ -39,10 +39,6 @@ Size after:
 
 **NKRO** がまだ動作しない場合は、`Magic` **N** コマンド(デフォルトでは `LShift+RShift+N`)を試してみてください。**NKRO** モードと **6KRO** モード間を一時的に切り替えるためにこのコマンドを使うことができます。**NKRO** が機能しない状況、特に BIOS の場合は **6KRO** モードに切り替える必要があります。
 
-ファームウェアを `BOOTMAGIC_ENABLE` でビルドした場合、`ブートマジック` **N** コマンドで切り替える必要があります(デフォルトでは `Space+N`)。この設定は EEPROM に格納され、電源を入れ直しても保持されます。
```

Comment:
This paragraph still seems to be in docs/faq_misc.md. Doesn't that also need to be updated?

Reply:
Updated in [`f4ae08c`](https://github.com/qmk/qmk_firmware/pull/15002/commits/f4ae08cffec5ccc07fa9ac5b1959f071ec9febf7)